scripts/send-email-to-unresolved-breach-subscribers.js

```diff
@@ -0,0 +1,23 @@
+const DB = require('../db/DB')
+const EmailUtils = require('../email-utils')
+const { LocaleUtils } = require('../locale-utils')
+
+async function sendUnresolvedBreachEmails () {
+  const subscribers = await DB.getSubscribersWithUnresolvedBreaches()
```

Comment:
risk: If we're planning to run this via `node scripts/send-email-to-unresolved-breach-subscribers.js`, then when this result set gets really big, we will want a way to send these emails in batches. If it's all done in a single command, that command process is going to take such a long time, and use up so much memory, that it will be hard to tell if it's actually working, or if it got hung up.

change (blocking): We should also include a way to track (a new DB field like `subscribers.needs_unresolved_breach_email`?) whether or not a subscriber has already been sent the email. (Are we sending these on a regular basis?) When there are errors during this process, we do NOT want to send repeat emails to people who have already received one.

Reply:
@groovecoder this has been updated, please review latest dif!
https://github.com/mozilla/blurts-server/blob/ca674258389ceca63dda0ad8095865fb7452a8ed/scripts/send-email-to-unresolved-breach-subscribers.js#L21
https://github.com/mozilla/blurts-server/blob/ca674258389ceca63dda0ad8095865fb7452a8ed/db/DB.js#L430-L442